src/Solver/Interoperability.cpp

```diff
@@ -369,6 +373,22 @@ seissol::Interoperability::~Interoperability()
   delete[] m_ltsFaceToMeshFace;
 }
 
+void seissol::Interoperability::setTravellingWaveInformation(const double* origin, const double* kVec, const double* ampField) {
+  assert(origin != nullptr);
+  assert(kVec != nullptr);
+  assert(ampField != nullptr);
+
+  m_travellingWaveParameters.origin = {origin[0], origin[1], origin[2]};
+  m_travellingWaveParameters.kVec = {kVec[0], kVec[1], kVec[2]};
+  constexpr double eps = 1e-15;
+  for (size_t i = 0; i < NUMBER_OF_QUANTITIES; i++) {
```

Comment:
this won't work with attenuation?

Reply:
Indeed, that's why I did not enable it for attenuation: https://github.com/SeisSol/SeisSol/blob/d08b8eb4171e938514d35a9ca32c2756e16f3af4/src/Solver/Interoperability.cpp#L948

Reply:
ok!